app/Http/Controllers/UserAddressController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Repositories\UserAddressRepositoryEloquent;
+use App\Http\Requests\CreateUserAddressRequest;
+use App\Services\UserAddressService;
+use App\Http\Requests\UpdateUserAddressRequest;
+use App\Http\Resources\UserAddressResource;
+
+class UserAddressController extends Controller
+{
+    protected $userAddressRepository;
+
+    public function __construct(UserAddressRepositoryEloquent $userAddressRepository)
+    {
+        $this->userAddressRepository = $userAddressRepository;
+    }
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $userAddres = $this->userAddressRepository->all();
```

Comment:
jangan all, karna kalau all dia akan menampilkan seluruh list alamat.
yang kita butuhkan kan hanya alamat user yang bersangkutan.
manfaatkan fitur $this->userAddressRepository->findWhere(['user_id'=>Auth::user()->id]); // silahkan sesuaikan